_docs/grader/grade_inquiry_grader.md

```diff
@@ -60,7 +60,7 @@ person during office hours.
 
 ### Grade Inquiry Per Component
 
-If grade inquiry per component is enabled, grade inquiries can be made for a specific component. For example, assume there is a component of the gradeable named Read Me and a component named Documentation. Now two separate grade inquiries can be made for each of these components. The grader(s) who graded that component will get notified if they are a grader that can be shown.
+If grade inquiry per component is enabled, grade inquiries can be made for a specific component. For example, assume there is a component of the gradeable named ReadMe and a component named Documentation. Now two separate grade inquiries can be made for each of these components. The grader(s) who graded that component will get notified if they are a grader that can be shown.
```

Comment:
I like this change, and let's further it and make it `README` which is the "official" capitalization of it (it even has a wikipedia page! https://en.wikipedia.org/wiki/README)